sdk/openai/azure-ai-openai-assistants/src/main/java/com/azure/ai/openai/assistants/models/AssistantsApiResponseFormatOption.java

```diff
@@ -71,6 +74,7 @@ public AssistantsApiResponseFormat getFormat() {
      *
      * @param responseFormatBinaryData input JSON string
      * @return a new instance of AssistantsApiResponseFormatOption
+     * @throws IllegalArgumentException if the provided JSON string does not match the expected format
```

Comment:
I think you might get some merge conflicts here, but they should be easy to address I hope. Sorry for giving you such a half-baked base feature branch 😓